docs/team/JosephLimWeiJie.md

```diff
@@ -1,6 +1,357 @@
-# John Doe - Project Portfolio Page
+# Joseph Lim Wei Jie - Project Portfolio Page
 
 ## Overview
 
+* [1. Restaurant Daily Report](#daily-report)
+* [2. Summary of Contributions](#summary-contribution)
+    + [2.1 Code contribution](#code-contribution)
+    + [2.2 Enhancements implemented](#enhancement-implemented)
+    + [2.3 Contributions to documentation](#contribute-documentation)
+    + [2.4 Contributions to the Developer Guide](#contribute-developerguide)
+    + [2.5 Contributions to team-based tasks](#contribute-team)
+    + [2.6 Review/Mentoring contributions](#review-mentoring)
+    + [2.7 Contributions beyond the project team](#contribute-team)
+* [3. Contributions to the User Guide](#contribute-ug)
+* [4. Contributions to the Developer Guide](#contribute-dg)
 
-### Summary of Contributions
+<a name="design-report"></a>
+
+### 1. Restaurant Daily Report
```

Comment:
It seems like the PPP will exceed the hard page limit (2 pages). 
Maybe you can consider briefly summarize?